cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -350,14 +350,6 @@ protected void configure() {
       }
     }
 
-    @Provides
-    @Named(Constants.AppFabric.SERVER_ADDRESS)
-    @SuppressWarnings("unused")
-    public InetAddress providesHostname(CConfiguration cConf) {
```

Comment:
moved it to config module, you suggested that here, https://github.com/caskdata/cdap/pull/5929#discussion_r69239278